libs/wire-api/src/Wire/API/Push/V2.hs

```diff
@@ -300,8 +299,8 @@ newPush from to pload =
       _pushIsCellsEvent = False
     }
 
-singletonPayload :: (ToJSONObject a) => a -> List1 Object
-singletonPayload = List1.singleton . toJSONObject
+singletonPayload :: (ToJSONObject a) => a -> NonEmpty Object
+singletonPayload a = toJSONObject a :| []
```

Comment:
There is a `singleton` for `NonEmpty` as well: https://hackage.haskell.org/package/base-4.21.0.0/docs/Data-List-NonEmpty.html#v:singleton

How about

```suggestion
singletonPayload = singleton . toJSONObject
```
(N.B. I haven't tried this. It type checking might fail.)